packages/dart/sshnoports/tools/Dockerfile.sshnpd-slim

```diff
@@ -0,0 +1,66 @@
+# To build using docker buildx
+# docker buildx build --push --platform linux/amd64,linux/arm64,linux/arm/v7 \
+# --tag <YOUR DOCKERHUB ID>/sshnpd-slim-multi:latest  \
+# -f packages/dart/sshnoports/tools/Dockerfile.sshnpd-slim .
+
+# Note dart:beta needs to be used when using M1 mac as known bug causes
```

Comment:
For M1 to AMD64 yes but there is another bug in Dart for M1 to Armv7

Reply:
So that make no difference, so we need to wait for Beta and for arm7 to work on M1/M2 macs
https://github.com/dart-lang/sdk/issues/54446#issuecomment-1876668624